nsq.h

```diff
@@ -78,6 +85,10 @@ void free_nsqd_connection(struct NSQDConnection *conn);
 int nsqd_connection_connect(struct NSQDConnection *conn);
 void nsqd_connection_disconnect(struct NSQDConnection *conn);
 
+struct NSQDConnInfo *new_nsqd_connection_status(struct NSQReader *rdr, const char *address, int port,
```

Comment:
our typical convention is to name this identical to the `struct` it returns, which would be `new_nsqd_conn_info`